### ISSUE TITLE
Support marker elements

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/LegacyMan#487]
There is a pattern where a coloured star is used to direct the reader to a footnote.

I'm pretty sure we handle this in the html to DITA for other instances - I that if the color is #F00 we set outputclass to `colorRed`

Content here:
https://github.com/DeepBlueCLtd/LegacyMan/pull/483/commits/421ea5b4112bc2e6f579226183a5d3d8f67f349a

## Update
Have inspected the data. We support this pattern when the color is specified in a `span`, but not when it is specified in a `strong`.

## Update 2
We do successfully handle `strong` elements, but we just need to correct the `outputclass` ID we use.

**Comment by robintw** [comment]
The `*` bit is being output in the DITA as: `<b outputclass="red">*</b>`.

This is from a part of the html to DITA code that converts `<strong>` elements with various styles to have various output classes:

```
        if strong.has_attr("style"):
            if "color:" in strong["style"]:
                if "#F00" in strong["style"]:
                    strong["outputclass"] = "red"
                elif "#00F" in strong["style"]:
                    strong["outputclass"] = "blue"
            del strong["style"]
```

However, later in the html to DITA code, it converts span with styles to have outputclass values like `colorRed` and `colorBlue`.

Should we only be using one of `red` or `colorRed`, and if so, which one?

Currently the star is _not_ showing up in the final HTML as red.

**Comment by IanMayo** [comment]
Thanks @robintw - we should be using the `colorRed` CSS class, as specified here:
https://github.com/DeepBlueCLtd/LegacyMan/blob/develop/template/F13ldMan/oxygen.css#L39

**Comment by IanMayo** [comment]
Aah, when `Wales_Composite.html` is properly processed, we do have the `outputclass` assigned, we just need to correct the ID of the class to `colorRed`.

![image](https://github.com/DeepBlueCLtd/LegacyMan/assets/1108513/051f842a-d0c1-43f7-9945-88e9c0354b9f)

**Comment by robintw** [comment]
Fixed in #492.
